sdk/keyvault/azcertificates/go.mod

```
@@ -1,12 +1,12 @@
 module github.com/Azure/azure-sdk-for-go/sdk/keyvault/azcertificates
 
-go 1.17
+go 1.18
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.22.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.13.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.9.1
-	github.com/Azure/azure-sdk-for-go/sdk/keyvault/internal v0.2.1
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.23.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.13.2
```

May as well upgrade to the latest
```suggestion
	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.14.0
```